cmake/thirdparty/FindCUDA.cmake

```diff
@@ -1844,6 +1830,8 @@ macro(CUDA_ADD_EXECUTABLE cuda_target)
   ####################
   if (NOT CUDA_LINK_WITH_NVCC)
       CUDA_COMPUTE_SEPARABLE_COMPILATION_OBJECT_FILE_NAME(link_file ${cuda_target} "${${cuda_target}_SEPARABLE_COMPILATION_OBJECTS}")
+  else()
+     set(link_file "")
   endif()
```

Comment:
hard to tell on quick review -- these stmts are already enclosed in BLT CHANGE comment blocks?

Reply:
yup